modules/nw-route-specific-annotations.adoc

```diff
@@ -188,3 +188,16 @@ The following table provides examples of the path rewriting behavior for various
 |/foo/|/foo/|/|/
 |/foo/|/foo/bar|/|/bar
 |===
+
+Certain special characters in `haproxy.router.openshift.io/rewrite-target` require special handling because they must be escaped properly. Refer to the following table to understand how these characters are handled.
+
+.Special character handling:
+[cols="3*", options="header"]
+|===
+|For character|Use characters|Notes
+|#|\#|Avoid # because it terminates the regular expression
+|%|% or %%|Avoid odd sequences such as %%%
+|‘| \’|Avoid ‘ because it is ignored
+|===
+
+All other characters can be used without escaping.
```

Comment:
I think we need to distinguish between characters and valid URL characters. We didn't include `\`, but that's not a valid URL character, so that's fair. But we should be clear:

```suggestion
All other valid URL characters can be used without escaping.
```